src/main/flight/mixer.c

```diff
@@ -477,6 +480,12 @@ void mixerConfigureOutput(void)
     loadLaunchControlMixer();
 #endif
     mixerResetDisarmedMotors();
+    
+    uint16_t checkFlags = 1;
+    for (uint16_t i = 0; i < motorCount; i++) {
```

Comment:
I am new to C so i will have to figure out which data types it supports natively and which are added by libraries.

Reply:
It's not about libraries .. Using variable with (bit)size smaller than CPU register forces compiler to emit additional instructions.

My personal view is that `int` is simply some variable(I'm lazy, it saves few keystrokes ;-) ), `signed int` / `signed` (identical to int) accents that variable may be negative and `unsigned` where unsigned value is expected (usually using it in combination with `size_t` /`sizeof` to avoid warning.
`uint<x>_t` stresses that variable width is important

Reply:
So what size is an int on this platform? 8 bit? 16 bit? 32 bit? If not specified otherwise is it signed or unsigned?
I assume the registers are 32 bit on a 32 bit platform. Are there fragments  of registers that save cycles or memory space (like AH, AX, EAX on a x86)?

Reply:
ARM is 32 bit.
`int` is signed (notice that `char` is neither signed nor unsigned)
ARM has instructions to sign-extend or zero-extend register from 8/16 bits (UXTB/UXTH/SXTB/SXTH). gcc does (maybe used to) emit these instruction even when not necessary.
ARM is RISC architecture, so most instructions take full registers as arguments, there is minimal support for packing multiple values into single register (there are SIMD/DSP instructions on Cortex M4 and Cortex M7)